webpack/scenes/Subscriptions/Details/SubscriptionDetailEnabledProducts.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router';
+import { notify } from '../../../move_to_foreman/foreman_toast_notifications';
+import { Table } from 'patternfly-react';
+import {
+  Table as ForemanTable,
+  TableBody as ForemanTableBody,
+  headerFormatter,
+  cellFormatter,
+  selectionHeaderCellFormatter,
+  collapseableAndSelectionCellFormatter,
+} from '../../../move_to_foreman/components/common/table';
+
+const SubscriptionDetailEnabledProducts = ({ enabledProducts }) => {
+  const column = (column_overrides = {}) => {
+    return {
+      property: '',
+      header: {
+        label: '',
+        formatters: [headerFormatter],
+      },
+      cell: {
+        formatters: [cellFormatter],
+      },
+      ...column_overrides,
+    }
+  };
+
+  const simple_column = (property, label) =>
+    column({
+      property: property,
+      header: {
+        label: __(label),
+        formatters: [headerFormatter],
+      },
+    });
+
+  const emptyStateData = () => ({
+    header: __('There are no Subscription Allocations to display'),
+    description: __('Subscription Allocations allow you to export subscriptions from the Red Hat Customer Portal to ' +
+        'an on-premise subscription management application such as Red Hat Satellite.'),
+    docUrl: 'http://redhat.com',
+    action: {
+      title: __('Import a Manifest to Begin'),
+      url: '/subscriptions',
+    },
+  });
+
+  const rows = () => {
+    return enabledProducts.results.flatMap(product =>
+      product.available_content.flatMap(content => {
+        return {
+          product: product.name,
+          enabled: content.enabled,
+          ...content.content,
+        };
+      })
+    );
+  };
+
+  const enabledProductColumns = [
```

Comment:
We have been putting these in schema files to make the component itself less complicated.  See  [`UpstreamSubscriptionsTableSchema`](https://github.com/Katello/katello/blob/master/webpack/scenes/Subscriptions/UpstreamSubscriptions/UpstreamSubscriptionsTableSchema.js) for example.